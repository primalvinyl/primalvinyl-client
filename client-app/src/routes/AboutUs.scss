@import '../styles/partials/breakpoints';
@import '../styles/partials/variables';

.about-us {
    padding: 2em;

    .wrapper {
        margin: 0 auto;

        @include breakpoint(lg-up){
            width: 1200px;
        }

        h2 {
            font-size: 20px;
            font-weight: bold;
        }
    }
}
