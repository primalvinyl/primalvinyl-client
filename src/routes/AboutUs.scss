@import '../styles/partials/breakpoints';
@import '../styles/partials/variables';

.about-us {
    .wrapper {
        margin: 0 auto;
        width: 1200px;

        h2 {
            font-size: 20px;
            font-weight: bold;
        }
    }

    @include breakpoint(lg-down){
        .wrapper {
            width: 100%;
        }
    }
}
