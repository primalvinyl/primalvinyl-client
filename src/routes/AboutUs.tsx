import React from 'react';
import MasterTemplate from '../components/MasterTemplate';
import './AboutUs.scss';

const AboutUs = (): React.ReactElement => {
    return (
        <MasterTemplate>
            <div className="about-us">
                <div className="wrapper">
                    <h2>About Us</h2>
                </div>
            </div>
        </MasterTemplate>
    );
};

export default AboutUs;
