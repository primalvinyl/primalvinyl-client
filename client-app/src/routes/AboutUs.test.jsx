import React from 'react';
import { shallow } from 'enzyme';
import Renderer from 'react-test-renderer';
import AboutUs from './AboutUs';

describe('AboutUs', () => {
    it('renders', () => {
        const myWrapper = shallow(<AboutUs />);
        expect(myWrapper.exists());
    });
    it('matches snapshot', () => {
        const component = Renderer.create(<AboutUs />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
