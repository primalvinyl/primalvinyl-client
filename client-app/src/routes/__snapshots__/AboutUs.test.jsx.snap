// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs matches snapshot 1`] = `
<div
  className="about-us"
>
  <div
    className="wrapper"
  >
    <h2>
      About Us
    </h2>
  </div>
</div>
`;
